### PR TITLE
added png creation for dot files in graphs folder to dodo

### PR DESCRIPTION
**Pull request by fa-me**

### CODE REVIEW

**Comment by Sh4pe** [comment]
Is the commit that changes `dodo.py` maybe missing here?

**Comment by fa-me** [comment]
added missing commit 21fb78d

**Review by Sh4pe — CHANGES_REQUESTED** [review]
(please ignore me)

**Review by Sh4pe — CHANGES_REQUESTED** [review]
I think that the generated `.png` files should not be contained in the repository. They are generated from the `.dot`s, so they don't contain new information and are quite large.

Please remove them from the repo and add something like `docs/graphs/png/*.png` to `.gitignore`.

File: dodo.py [hunk 1]
```diff
@@ -12,8 +14,22 @@ def integration_test_action():
             'actions': [action]
         }
 
+def task_graphviz():
+    graph_dir = "docs\\graphs\\"
```
Comment (Sh4pe) [inline dodo.py]: The `\` as path separator only works on Windows, my machine does nothing when I `doit graphviz`.

I guess a more portable solution would be to set `graph_dir = ['docs', 'graphs']` and then call

```
os.path.join(*graph_dir, ...)
```

below in lines 19 and 25.

**Review by Sh4pe — CHANGES_REQUESTED** [review]

**Review by Sh4pe — CHANGES_REQUESTED** [review]
Just trying to understand GitHub. This comment is irrelevant.

File: dodo.py (hunk 1 above)
Comment (Sh4pe) [reply dodo.py]: Great, your solution is better than my proposal

**Review by Sh4pe — APPROVED** [review]

**Pull request merged**